nautobot_ssot/integrations/servicenow/diffsync/adapter_nautobot.py

```diff
@@ -127,7 +127,7 @@ def load(self):
 
     def tag_involved_objects(self, target):
         """Tag all objects that were successfully synced to the target."""
-        # The ssot-synced-to-servicenow tag *should* have been created automatically during plugin installation
+        # The servicenow_last_synchronized tag *should* have been created automatically during plugin installation
```

Comment:
Why isn't this also using the `ssot_last_synchronized` and `system_of_record` cfs?

Reply:
Hm, then maybe something like `last_synced_from_ssot` instead of `ssot_last_synchronized` would be clearer?

Reply:
I was using `ssot_last_synchronized` as that's what I've used in other projects but that suggestion is slightly shorter and yes I can see it being clearer.